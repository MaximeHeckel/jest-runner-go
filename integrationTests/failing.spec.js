const run = require('./run');

it('Works when it has failing tests', () => {
  return expect(run('failing')).resolves.toMatchSnapshot();
});