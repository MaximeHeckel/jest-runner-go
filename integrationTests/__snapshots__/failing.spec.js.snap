// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has failing tests 1`] = `
"FAIL integrationTests/fixtures/failing/main_test.go
  âœ• /go/src/project/go-test-runner/integrationTests/fixtures/failing/main_test.go
	main_test.go:9: Got 5 expected 6
Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Ran all test suites.
"
`;
